Chart.yaml

```diff
@@ -26,3 +26,20 @@ details:
   This Helm chart is a fully installable app that codifies StackStorm cluster optimized for HA and K8s environment.
   By default FOSS community version of st2 will be installed. Enterprise version can be enabled as an option.
   For configuration details check 'values.yaml'.
+dependencies:
+  - name: rabbitmq
+    version: 8.0.2
+    repository: https://charts.bitnami.com/bitnami
+    condition: rabbitmq.enabled
+  - name: mongodb
+    version: 10.0.1
```

Comment:
Should we consider 8.0.0 instead considering the bitnami chart for mongo with version 9.0.0 looks to have brought mongodb 4.4.x support? This version is not certified with StackStorm and there were some issues reported around 4.2.x. 
cc: @armab

Reply:
Right, StackStorm supports maximum mongodb 4.0.
And because we pin the version here:
https://github.com/StackStorm/stackstorm-ha/blob/47dae4971fe0b1512758ce6268d2cf4126bae00b/values.yaml#L435-L437
it should be OK